app/views/v2/search-crn/search-crn.njk

```diff
@@ -20,7 +20,40 @@
   <div class="govuk-grid-column-two-thirds">
     <form method="POST" action="/v2/search-crn">
       <input type="hidden" name="crumb" value="{{crumb}}"/>
-      {{ govukInput(model) }}
+
+      {% if errors %}        
+        <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+          <h2 class="govuk-error-summary__title" id="error-summary-title">
+            There is a problem
+          </h2>
+          <div class="govuk-error-summary__body">
+            <ul class="govuk-list govuk-error-summary__list">
+              {% for errorDetail in errors.details %}
+                <span class="govuk-error-message">
+                  <span class="govuk-visually-hidden">Error:</span> {{ errorDetail.message }}
+                </span>
+              {% endfor %}
+            </ul>
+          </div>
+        </div>        
+      {% endif %}
+
+
+      <div class="govuk-form-group">
+        <h1 class="govuk-label-wrapper"><label class="govuk-label govuk-label--l" for="event-name">
```

Comment:
Maybe change the format of these input boxes to match the "asking multiple questions on a page" section here -> https://design-system.service.gov.uk/components/text-input/

Reply:
Changed headings